src/neuroconv/datainterfaces/behavior/video/videodatainterface.py

```diff
@@ -23,7 +23,7 @@ class VideoInterface(BaseDataInterface):
     """Data interface for writing videos as ImageSeries."""
 
     def __init__(
-        self, file_paths: list[str | Path], verbose: bool = False
+        self, file_paths: list, verbose: bool = False
```

Comment:
Supporting the ability for people to call `from __future__ import annotations` in the library is one thing; adding full support for schema inference from data interface signatures of the updated kind is another, and would in preferance be done with Pydantic

Removing for now